tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/report.py

```diff
@@ -0,0 +1,40 @@
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+from gaiatest.form_controls.header import GaiaHeader
+from gaiatest.tests.functional.messages.test_sms_report import TestSmsReport
+
+class Report(Base):
+
+    _time_sent_locator = (By.CSS_SELECTOR, 'span.sent-timestamp')
+    _time_received_locator = (By.CSS_SELECTOR, 'span.received-timestamp')
+
+    _header_locator = (By.ID, 'information-report-header')
+
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
+
+    def close_report(self):
+        GaiaHeader(self.marionette, self._header_locator).go_back()
+        from gaiatest.apps.messages.regions.message_thread import MessageThread
+        return MessageThread(self.marionette) 
+
+    def is_message_an_sms(self):
+        return self.marionette.find_element(*_text_type).get_attribute('data-l10n-id') == 'message-type-sms'
+
+    def is_date_sent_ok(self):
+        time_sent = TestSmsReport.time_sent
+        sent_date = self.marionette.find_element(*_time_sent_locator).get_attribute('data-l10n-date')
+        
+        time_sent_err = (time_sent + 1200) #Two minutes error
```

Comment:
Representing 2 minutes with 1200, looks weird. Are 1200 in seconds? In milliseconds? In tenth of seconds?

In order to prevent the future reader to hesitate, we can name a variable:
``` py
error_range_in_tenth_of_seconds: 1200
```